x/concentrated-liquidity/tick_test.go

```diff
@@ -1076,6 +1076,10 @@ func (s *KeeperTestSuite) TestGetTickLiquidityNetInDirection() {
 			startTick:       sdk.NewInt(10),
 			boundTick:       sdk.NewInt(-15),
 			expectedLiquidityDepths: []queryproto.TickLiquidityNet{
+				{
+					LiquidityNet: sdk.NewDec(-20),
+					TickIndex:    10,
+				},
```

Comment:
I don;t understand why this is returned even with the bound tick set to -15?
Is your proposal here to change this query to return whatever is provided by bound tick + 1 tick?